### PR TITLE
ENH 11 rc1

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {% set version = "10.0.1" %}
```
Comment (beckermr) [inline recipe/meta.yaml:1]:
```suggestion
{% set version = "11.0.0.rc1" %}
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by isuruf** [comment]
Pushed to `rc` branch

**Pull request merged**